endpoints/apiserving.py

```diff
@@ -183,6 +184,90 @@ def _get_app_revision(environ=None):
     return environ['CURRENT_VERSION_ID'].split('.')[1]
 
 
+class ApiConfigRegistry(object):
+  """Registry of active APIs"""
+
+  def __init__(self):
+    # Set of API classes that have been registered.
+    self.__registered_classes = set()
```

Comment:
Any reason these attributes are "private"?

Reply:
That's just the way they've been for as long as I've been here.

Reply:
(This class isn't new; it's just moved.)